src/app/pages/token/token-detail/token-content/token-content-tab/token-transfers-tab/token-transfers-tab.component.ts

```diff
@@ -261,12 +273,75 @@ export class TokenTransfersTabComponent implements OnInit, AfterViewInit {
     });
   }
 
+  async getListTransactionTokenIBC(nextKey = null, isReload = false) {
+    const denomFilter = this.tokenDetail.channelPath?.path + '/' + this.tokenDetail.channelPath?.base_denom;
+    let payload = {
+      limit: this.pageData.pageSize,
+      heightLT: nextKey,
+      denom: denomFilter,
+      offset: this.pageData.pageIndex * this.pageData.pageSize,
+      address: null,
+    };
+
+    if (this.keyWord) {
+      if (this.keyWord?.length === LENGTH_CHARACTER.TRANSACTION && this.keyWord == this?.keyWord.toUpperCase()) {
+        payload['txHash'] = this.keyWord;
+      } else {
+        payload['address'] = this.keyWord;
+      }
+    }
+
+    this.tokenService.getListTransactionTokenIBC(payload).subscribe({
+      next: (res) => {
+        if (res) {
+          this.nextKey = null;
+          if (res.ibc_ics20.length >= 100) {
+            this.nextKey = res?.ibc_ics20[res.ibc_ics20.length - 1]?.height;
+            this.hasMore.emit(true);
```

Comment:
Check lại `this.hasMore`, hình như không còn sử dụng

Reply:
vẫn còn sử dụng overview
![image](https://github.com/aura-nw/aurascan/assets/99626599/aa071366-35e3-4fbe-8363-b9446ade499b)